### DISCUSSION TITLE
Noob developer

### DISCUSSION THREAD

**Discussion by souhailmay** [GSI-CS-CO/chart-fx#612] [Q&A]
Hello team,

I apologize if my question seems basic, but I'm a complete beginner in Java programming and I'm interested in utilizing your library. Specifically, I'd like to create a real-time candlestick chart. However, when I examined the example, I couldn't quite grasp it. Could someone please provide me with a straightforward example of how to add a candlestick chart using my own data (Bybit Api)?

**Comment by yezhengli-Mr9** [comment]
> Hello team,
> 
> I apologize if my question seems basic, but I'm a complete beginner in Java programming and I'm interested in utilizing your library. Specifically, I'd like to create a real-time candlestick chart. However, when I examined the example, I couldn't quite grasp it. Could someone please provide me with a straightforward example of how to add a candlestick chart using my own data (Bybit Api)?

I successfully applied to my own data within one week (while I have to work for major hours). Feel https://github.com/fair-acc/chart-fx/blob/ed86318c5a1bde23d285633a94860648aee5fb32/chartfx-samples/src/main/java/io/fair_acc/sample/financial/FinancialRealtimeCandlestickSample.java#L68
are good samples for beginners like me. My sample output is here https://github.com/fair-acc/chart-fx/discussions/641 although I have many features to integrated into for the upcoming near future.
